tests/test_base.py

```diff
@@ -4,56 +4,66 @@
 except ImportError:
     from http.client import OK
 
-from mock import patch
-from tests import TestCase
+import json
+import requests
+import requests_mock
 
+from tests import TestCase
```

Comment:
I wish I would use pytest for testing
But fine for now. I may find some time to update tests